server/utils/oasysImportUtils.test.ts

```diff
@@ -100,4 +100,15 @@ describe('OASysImportUtils', () => {
       expect(fetchOptionalOasysSections(application)).toEqual([1, 2])
     })
   })
+
+  describe('sortOasysImportSummaries', () => {
+    it('sorts the imports into order of questions', () => {
+      const oasysSummary1 = roshSummaryFactory.build({ questionNumber: '3' })
+      const oasysSummary2 = roshSummaryFactory.build({ questionNumber: '2' })
+      const oasysSummary3 = roshSummaryFactory.build({ questionNumber: '1' })
```

Comment:
This tripped me up a little. I think I'd prefer 

```
const oasysSummary1 = roshSummaryFactory.build({ questionNumber: '1' })
```
etc., then have `[oasysSummary3, oasysSummary2, oasysSummary1]` going in and `[oasysSummary1, oasysSummary2, oasysSummary3]` coming out. Though this is very much personal aesthetic preference